kubernetes/schema_pod_spec.go

```diff
@@ -344,6 +344,30 @@ func podSpecFields(isUpdatable, isComputed bool) map[string]*schema.Schema {
 							Type: schema.TypeInt,
 						},
 					},
+					"windows_options": {
+						Type:        schema.TypeSet,
```

Comment:
Let's make this a `TypeList` and set `MaxItems: 1` to follow the convention that's been used for other nested blocks in the pod spec.